<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlayClass</class>
 <widget class="QWidget" name="XPlayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlay</string>
  </property>
  <widget class="XVideoWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
  </widget>
  <widget class="XSlider" name="playPos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>1241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>600</y>
     <width>115</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>XVideoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>XSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XPlay.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playPos</sender>
   <signal>sliderPressed()</signal>
   <receiver>XPlayClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>967</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPos</sender>
   <signal>sliderReleased()</signal>
   <receiver>XPlayClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>820</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>XPlayClass</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>837</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>play</sender>
   <signal>clicked()</signal>
   <receiver>XPlayClass</receiver>
   <slot>playOrpause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>644</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>892</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFile()</slot>
  <slot>playOrpause()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
 </slots>
</ui>
